### PR TITLE
Update the README.

### PR DESCRIPTION
**Pull request by carlosalberto**
Include Release Schedule and Snapshots info.

(Removed most of the Plans section, as it's not needed anymore IHMO)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,53 +24,61 @@ We would love to hear from the larger community: please provide feedback proacti
...
+    <repository>
+      <id>oss.sonatype.org-snapshot</id>
+      <url>https://oss.jfrog.org/artifactory/oss-snapshot-local</url>
+    </repository>
+  </repositories>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-sdk</artifactId>
```
Comment (songy23) [inline README.md]:
```suggestion
      <artifactId>opentelemetry-api</artifactId>
```

File: README.md [hunk 2]
```diff
@@ -24,53 +24,61 @@ We would love to hear from the larger community: please provide feedback proacti
...
+
+### Gradle
+
+```groovy
+repositories {
+	maven { url 'https://oss.jfrog.org/artifactory/oss-snapshot-local' }
+}
+
+dependencies {
+	compile('io.opentelemetry:opentelemetry-sdk:0.1.0-SNAPSHOT')
```
Comment (songy23) [inline README.md]:
```suggestion
	compile('io.opentelemetry:opentelemetry-api:0.1.0-SNAPSHOT')
```

Comment (tylerbenson) [reply README.md]: Should we list both dependencies?

Comment (songy23) [reply README.md]: We can mention that libraries probably only need to depend on APIs, while applications need both. See https://github.com/census-instrumentation/opencensus-java#opencensus-quickstart-for-libraries and https://github.com/census-instrumentation/opencensus-java#opencensus-quickstart-for-applications.

**Comment by carlosalberto** [comment]
Hey, thanks for the feedback. Was wondering about which artifact(s) to list. I'm happy to put `opentelemetry-api` as the default one, with a small note about it (maybe listing what 1-2 of the other important artifacts).

**Comment by songy23** [comment]
> I'm happy to put opentelemetry-api as the default one, with a small note about it (maybe listing what 1-2 of the other important artifacts).

This sounds good to me.

**Comment by carlosalberto** [comment]
It seems javadocs couldn't get generated and thus the build failed (which, btw, is not related to the README changes here): https://circleci.com/gh/open-telemetry/opentelemetry-java/487?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Any idea on this?

**Comment by carlosalberto** [comment]
Never mind, merging the upstream changes fixed that. Merging ;)

**Pull request merged**